platforms/config.default.json

```diff
@@ -37,13 +37,17 @@
   },
   "queue": {
     "timeout": 10000,
-    "retries": 1000,
+    "retries": 6,
+    "backoffTime": 120000,
+    "backoffStrategy": "fixed",
     "defaults": {
       "prefix": "notification",
       "stallInterval": 5000,
       "nearTermWindow": 1200000,
       "delayedDebounce": 1000,
       "redis": {
+        "host": "127.0.0.1",
```

Comment:
Is localhost a good default address?

Reply:
It is also used in live system and it is implicit set with the same value. I only add it to show that we can change it at this position.